Zend/zend_exceptions.stub.php

```diff
@@ -48,7 +48,7 @@ public function __toString(): string {}
 
 class ErrorException extends Exception
 {
-    public function __construct(string $message = UNKNOWN, int $code = 0, int $severity = E_ERROR, string $filename = UNKNOWN, int $lineno = 0, ?Throwable $previous = null) {}
+    public function __construct(string $message = "", int $code = 0, int $severity = E_ERROR, ?string $filename = null, ?int $line_number = null, ?Throwable $previous = null) {}
```

Comment:
```suggestion
    public function __construct(string $message = "", int $code = 0, int $severity = E_ERROR, ?string $filename = null, ?int $line = null, ?Throwable $previous = null) {}
```
as the method is called getLine().

Reply:
Renamed!